a10_octavia/controller/worker/controller_worker.py

```diff
@@ -346,7 +346,7 @@ def update_listener(self, listener_id, listener_updates):
         stop=tenacity.stop_after_attempt(RETRY_ATTEMPTS))
     def create_load_balancer(self, load_balancer_id, flavor=None, ctx_map=None, ctx_lock=None):
         """Function to create load balancer for A10 provider"""
-
+        time.sleep(60)
```

Comment:
Don't use sleep(60) but use a general API to check/re-check and wait instead.

Reply:
Fixed